api/ereqs_admin/static/admin/override.scss

```diff
@@ -46,3 +50,139 @@ $nav-menu-width: 200px;
   position: absolute;
   width: $nav-menu-width;
 }
+
+.app-admin_interface,
+.admin-interface {
+  background: $color-very-lt-gray;
+}
+
+.object-tools a {
+  border-radius: 3px;
+  background: $color-lt-gray;
+  color: $color-gray;
+  border: 1px solid $color-gray;
+  text-transform: none;
+}
+
+#content-main .submit-row input.default {
+  text-transform: none;
+}
+
+.app-admin_interface .form-row {
+  background: $color-white;
+}
+
+.module {
+  background: none;
+}
+
+#content  h1 {
+  font-weight: 500;
+  color: $color-black;
+  font-size: $space-lg;
```

Comment:
Rather than use the `space` variables for font size, let's use (and add) mixins from `type-size`. We'll generally want to set the line height whenever we're setting font size. Setting the line height will also likely resolve some of the need for other padding.